include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -488,7 +488,7 @@ ERROR(error_caching_no_cas_fs, none,
 ERROR(error_prefix_mapping, none, "cannot create scanner prefix mapping: '%0'", (StringRef))
 
 REMARK(replay_output, none, "replay output file '%0': key '%1'", (StringRef, StringRef))
-REMARK(output_cache_miss, none, "cache miss output file '%0': key '%1'", (StringRef, StringRef))
+REMARK(output_cache_miss, none, "cache miss output for input file '%0': key '%1'", (StringRef, StringRef))
```

Comment:
Is this the primary "cache miss" remark that will be seen? It might be clearer to just say "cache miss for input file" without mentioning "output" at all.